dev.aunetx.deezer.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>dev.aunetx.deezer</id>
+  <provides>
+    <id>dev.aunetx.deezer.desktop</id>
+  </provides>
+  <launchable type="desktop-id">dev.aunetx.deezer.desktop</launchable>
+  <name>Deezer</name>
+  <developer_name>aunetx</developer_name>
+  <summary>Online music streaming service</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.deezer.com/legal/cgu/</project_license>
```

Comment:
But then the readme says:
> Installing/using this is consequently outside of the scope of the Deezer EULA, and I am not responsible for your usage of this.

Reply:
Yes, I don't know how to do it (hence why I marked as draft). I messaged Deezer ([link in french](https://fr.deezercommunity.com/application-ordinateur-et-site-web-58/mettre-en-ligne-l-application-deezer-sur-flathub-pour-linux-40620)), so maybe if I have a positive response I will be able to remove the notice in the readme, as the project would be endorsed, and then I could link the EULA as needed.